### PR TITLE
added missing template

### PR DESCRIPTION
**Pull request by stefonarch**
Closes https://github.com/lxqt/qps/issues/266 and needed for weblate

### CODE REVIEW

**Comment by stefonarch** [comment]
I just noticed all the old translations in qps.desktop.in, they have to be moved to *.desktop.yaml files.

File: src/translations/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+project(qps)
```
Comment (palinek) [inline src/translations/CMakeLists.txt:1]: What is this CMakeLists files good for?

Comment (stefonarch) [reply src/translations/CMakeLists.txt:1]: Without it it didn't include the translations in qps.desktop, only the template.
Similar files are present  in other foo/translations directories.

Comment (palinek) [reply src/translations/CMakeLists.txt:1]: If I'm not mistaken, this was for building when the translations were built in lxqt-l10n. It definitely isn't needed for the `lxqt_translate_desktop`.

Comment (stefonarch) [reply src/translations/CMakeLists.txt:1]: I'll retest on debian 32bit, I know little about cmake.
So those could be removed from all translation folders?

Comment (stefonarch) [reply src/translations/CMakeLists.txt]: retested, don't know what happened yesterday, it's not needed. Doesn't make sense also.
Removed the file.

**Comment by stefonarch** [comment]
So can this be merged now?

**Comment by tsujan** [comment]
Merge it if you've tested it and it works fine.

**Pull request merged**